admin_scripts/ubuntu/ubuntu_16.04/upgrading/upgrade.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+
+export DEBIAN_FRONTEND=noninteractive
+apt-get update && apt-get -y upgrade
```

Comment:
This looks a bit like _System - Opdatér alt_, but without the error recovery. (But with the right `DEBIAN_FRONTEND` magic...) Should this one be removed?